docs/user_guide/02b_query_engine.md

```diff
@@ -149,42 +149,36 @@ lines: 41-56
 :::
 
 :::{note}
-Query methods used for changing the operands cannot be concatenated or assigned to variables, since their Return is None. That is, the following code snippet will set `gender_lowercase` as None, and as a result, an AttributeError will be thrown:
+Query methods used for changing the operands cannot be assigned to variables for further querying, since their return type is `None`. That can be appreciated in the following code snippet, where the variable `gender_lowercase` in the query takes None as value. An `AttributeError` is thrown as a consequence when trying to further query with the `equal_to` querying method:
```

Comment:
```suggestion
Query methods used for changing the operands cannot be assigned to variables for further querying, since their return type is `None`. The following code snippet shows an example, where the variable `gender_lowercase` evaluates to None. An `AttributeError` is thrown as a consequence when trying to further query with the `equal_to` querying method:
```